vars.go

```diff
@@ -121,30 +123,52 @@ func AddVulnerability(db *sql.DB, vuln *Vulnerability) error {
 	if rows, _ := res.RowsAffected(); rows < 1 {
 		errs.append(noRowsInserted, "AddVulnerability")
 	}
-	if e := SetCvss(tx, vuln); e != nil {
-		if !IsNoRowsError(e) {
-			return e
+
+	var e interface{}
+	// Setting values in the impact table
+	e = SetImpact(tx, vuln)
+	if ve, ok := e.(Err); ok {
+		if !ve.IsNoRowsError() {
+			return ve
 		}
-		errs.appendFromError(e, "AddVulnerability")
+		errs.appendFromError(ve.err, append([]string{"AddVulnerability"}, ve.parents...)...)
```

Comment:
but calling
`errs.appendFromError(ve.err, "AddVulnerability")`

would have the parents as only "AddVulnerability" right? Because
`func (es Errs) appendFromError(err error, parents ...string) { 
es = append(es, newErrFromErr(err, parents...)) 
}`
only "AddVulnerability" which is passed in as parents would get passed to newErrFromErr .. and even though it does a type assertion on the err passed in as an argument it would always be just an error not a varsErr right bc in the initial call we are passing in ve.err which is an actual error so it wouldn't have a parents variable?